snuba/migrations/parse_schema.py

```diff
@@ -44,7 +44,7 @@
     enum_str         = ~r"([a-zA-Z0-9\-]+)"
     enum_val         = ~r"\d+"
     agg              = "AggregateFunction" open_paren space* agg_func space* comma space* agg_types space* close_paren
-    agg_func         = ~r"[a-zA-Z]+\([a-zA-Z0-9\,\.\s]+\)|[a-zA-Z]+"
+    agg_func         = ~r"[a-zA-Z0-9]+\([a-zA-Z0-9\,\.\s]+\)|[a-zA-Z0-9]+"
```

Comment:
`uniqCombined64`

Reply:
Why do you need the first 0-9?

Reply:
To capture both `uniqCombined64` and `whatever64(with parameters)`
The first one is not the beginning of the string.